config/config.go

```diff
@@ -887,6 +888,7 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("analytics.pubstack.buffers.timeout", "900s")
 	v.SetDefault("amp_timeout_adjustment_ms", 0)
 	v.BindEnv("gdpr.default_value")
+	v.SetDefault("gdpr.default_value", 1)
```

Comment:
Please remove this line and set this value using env variables or other config type.

Reply:
in next commit i have solved:

The line has been removed